### PR TITLE
Add rename exports function to API

### PR DESCRIPTION
**Pull request by tpjanssen**
1) Added function to rename an export
2) Updated docs for missing delete export function
3) Possibility to refer to export without .mp4 extension

### CODE REVIEW

File: frigate/http.py [hunk 1]
```diff
@@ -1954,9 +1955,62 @@ def export_recording(camera_name: str, start_time, end_time):
     )
 
 
+def export_filename_check_extension(filename: str):
+    filename_split = os.path.splitext(filename)
+    if filename_split[1] == ".mp4":
```
Comment (NickM-27) [inline frigate/http.py:1960]: This should check split[-1] to catch cases where . is in the name

Comment (tpjanssen) [reply frigate/http.py:1960]: What do you exaclty mean Nick? Is it a problem when the filename contains a period character?

Comment (NickM-27) [reply frigate/http.py:1960]: Yes, since we know the file extension will come after the last . And we want to avoid issues where file name includes .

Comment (tpjanssen) [reply frigate/http.py:1960]: In that case a simple `filename.endswith(".mp4")` and a change from `re.search(r'[^.:_A-Za-z0-9]'` to `re.search(r'[^:_A-Za-z0-9]'` will do as well?

Comment (NickM-27) [reply frigate/http.py:1960]: Yes, though I'm not 100% sure what the regex is checking for

Comment (tpjanssen) [reply frigate/http.py:1960]: To make sure that we only allow alphanumeric characters, along with : and _ characters in the new filename.

**Review by NickM-27 — APPROVED** [review]

**Pull request merged**